src/utils/supports.js

```diff
@@ -7,6 +7,7 @@ import { getCurrentInstance } from './store-utils';
 // Non-semver(?) UA string detection
 const containPixelfed = /pixelfed/i;
 const notContainPixelfed = /^(?!.*pixelfed).*$/i;
+const containAnyOma = /(?:pler|akk)oma/i
```

Comment:
I'm not sure how's the pleroma forks landscape looks like. Would it be safer or forward-thinking if they are separate? e.g. `containPleroma` and `containAkkoma`?

Reply:
I don't think any other forks are going to spawn anytime soon, development is already pretty slow... :thinking:

Reply:
I think the variable name is kinda too "playful" 😉. It gives the impression that Phanpy will support _all_ *oma forks. And the name will clash if one day someone creates a new fedi server called 'Oma' 😂

Reply:
Fair enough :laughing: 

What would you recommend?

Reply:
I think it's safer to make it explicit, with both `containPleroma` and `containAkkoma`, so it'll have two `supports('@pleroma/local-visibility-post')` and `supports('@akkoma/local-visibility-post')`. This way, we can't assume that Akkoma fork is always compatible with Pleroma, and we could add additional `@akkoma/*` features down the line (if Akkoma is more developed than Pleroma or vice versa) — this logic can apply to _any_ fedi servers in the future.